contacts_vs_distance.ipynb

```diff
@@ -44,7 +44,7 @@
    "outputs": [],
```

Comment:
<p>Impact of sequencing depth on computed P(s)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open2c/open2c_examples/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>